Microsoft.Toolkit.Uwp/Helpers/HttpHelper/HttpHelper.cs

```diff
@@ -104,6 +109,45 @@ public HttpHelper(int poolSize, IHttpFilter httpFilter)
             }
         }
 
+        public async Task<HttpHelperResponse> GetRequestAsync(HttpHelperRequest request, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Sorry, I'm not sure to understand well your thought. You mean, **Twitter Service** should set HttpClient's default headers instead of **HttpHelper** ?

More exactly: the **ExecuteGetStreamAsync** method of **TwitterOAuthRequest** class should set Default Headers of a local HttpHelper instance ?